dataStandard_versioned_docs/version-5/whats-new/whats-new-v52.md

```diff
@@ -0,0 +1,204 @@
+
+_Assessment Accommodations_ entail altering the design or delivery of assessments to cater to the needs of students with disabilities or English learners (EL) who cannot effectively take standard assessments. These accommodations can differ depending on the assessment part, such as using a translator for ELA or a calculator for Mathematics. Some special education students may need additional modifications to participate in assessments.
+
+Use cases Ed-Fi has discussed with the community members revealed that a typical process of assessment registration in the field has following steps
+
+* Education organizations collaborate with the assessment vendor to determine the time frame and guidelines for administering an assessment.
+* The specifics of assessment participation and delivery are outlined.
+* Students are registered for the administration of the assessment.
+* The assessment vendor receives the registration list of students along with selected student information.
+
+#### Model Details
+
+To allow community members track data in the Ed-Fi UDM, Assessment Registration domain is created with the following entities and associations:
+
+<u>_AssessmentAdministration_:</u> It is to store data for the administration of an assessment. It has references to AssigningEducationOrganization and Assessment entities, and tracks information relating to AdministrationIdentifier, AssessmentAdministrationPeriod and AssessmentBatteryPart.
+
+<u>_AssessmentAdministrationParticipation_:</u> The planned participation of an EducationOrganization in the administration of an assessment.
+StudentAssessmentRegistration: Identifies a registration that indicates the student is expected to participate in a particular assessment administration.
+
+<u>_AssessmentBatteryPart_:</u> The parts organized for administering an assessment which together provide a comprehensive assessment of the students.
+StudentAssessmentRegistrationBatteryPartAssociation: The association to the part(s) of the assessment battery that the student is to be tested for this administration of the assessment.
+
+<u>_StudentEducationOrganizationAssessmentAccommodation_:</u> The accommodation(s) required or expected for administering assessments as determined by the education organization.
+
+The UML diagram below illustrates the model for the Assessment Registration domain with its entities, attributes and associations.
+
+![Assessment Registration Domain](https://edfidocs.blob.core.windows.net/$web/img/reference/data-standard/v52/UML_AssessmentRegistrationDomain_DSv52.jpg)
+[_Click here for a larger view of the diagram_](https://edfidocs.blob.core.windows.net/$web/img/reference/data-standard/v52/UML_AssessmentRegistrationDomain_DSv52.jpg)
```

Comment:
Best to have a line break between lines 62 and 63.  The "larger version" is typically a large image. Many of the images previously created from Visio are very large, much larger than what is needed for display in a web page. I see that this file is 375 KB, which is big but not gigantic (previously, some of the images were over 1 MB). We might want to make a small modification later on, but this is good enough for now. No change needed. And thank you for getting the images uploaded.

Reply:
Thanks and sure. I took the liberty to create the diagram image as a JPEG as opposed to BitMap or even GIF. I believe that is the main reason how file size reduced.